docs/gameplay-multiplayer-server-multiplayer.md

```diff
@@ -1004,7 +1004,7 @@ _Parameters_
 | dispatcher | [Dispatcher](#match-runtime-api-go) exposes useful functions to the match, and may be used by the server to send messages to the participants of the match. |
 | tick | Tick is the current match tick number, starts at 0 and increments after every `MatchLoop` call. Does not increment with calls to `MatchJoinAttempt`, `MatchJoin`, or `MatchLeave`. |
 | state | Custom match state interface, use this to manage the state of your game. You may choose any structure for this interface depending on your game's needs. |
-| graceSeconds | Messages is a list of data messages received from users between the previous and current tick. |
+| graceSeconds | The number of seconds provided to complete a graceful termination before a match is forcefully closed. |
```

Comment:
Please review/replace - I had a quick go at the parameter definition here.

Reply:
Thanks 🙏